modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js

```diff
@@ -390,7 +388,6 @@ function FieldBase({
 
 			{!hideEditedFlag && (
 				<input
```

Comment:
Could you explain why did you remove the key parameter here?

Reply:
We should only use the `key` attribute for element into an iteration (like an array)

The React documentation better explain that:

> A “key” is a special string attribute you need to include when creating lists of elements.

> Keys help React identify which items have changed, are added, or are removed. Keys should be given to the elements inside the array to give the elements a stable identity

https://reactjs.org/docs/lists-and-keys.html